pkg/pillar/cmd/vaultmgr/vaultmgr.go

```diff
@@ -464,50 +484,62 @@ func initializeSelfPublishHandles(ps *pubsub.PubSub, ctx *vaultMgrContext) {
 	}
 	pubVaultStatus.ClearRestarted()
 	ctx.pubVaultStatus = pubVaultStatus
+
+	//to publish sealed vault key to controller
```

Comment:
Comment seems incorrect since we're not publishing something sealed. Where publishing something encrypted AFAIK.

Reply:
Corrected it